sources/ascwp/src/main.rs

```diff
@@ -0,0 +1,59 @@
+mod config;
+mod error;
+use crate::config::Config;
+use crate::error::Result;
+use imdsclient::ImdsClient;
+use snafu::{OptionExt, ResultExt};
+use std::fs;
+use log::{error, info, warn};
+use std::time::Duration;
+use async_std::task;
+
+
+
+//Path to config file containing host's eks_autoscaling.should-wait setting
```

Comment:
For non docstring comments as well, I'd add a space between `//` and the start of the comment.